### PR TITLE
add test code

### PR DESCRIPTION
**Pull request by yingjingyang**

### CODE REVIEW

**Review by Loverush — APPROVED** [review]

File: basic/38-Flashbots-and-Eden/scripts/send-flashbots.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  let userNonce = await signerWallet.getTransactionCount();
+
+  console.log("userNonce: ",userNonce)
+
+  // Config the transaction
+  const contractTransaction = {
+    to: '0x87eCbC961193e833cB70B97E98053a76Ae458B94',
+    gasPrice: 200,
+    data: contractPayload,
+    nonce: 15
```
Comment (liberhe) [inline basic/38-Flashbots-and-Eden/scripts/send-flashbots.js:74]: nonce is fixed?

**Review by liberhe — APPROVED** [review]

File: basic/38-Flashbots-and-Eden/scripts/send-flashbots.js (hunk 1 above)
Comment (yingjingyang) [reply basic/38-Flashbots-and-Eden/scripts/send-flashbots.js:74]: Nonce auto used by etherjs will fail

File: basic/38-Flashbots-and-Eden/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "38-Flashbots-and-Eden",
+  "version": "1.0.0",
+  "description": "## 参考链接   flashbots github: https://github.com/flashbots/pm#resources   Eden Network 介绍: https://imtoken.fans/t/topic/41713    Eden Network 官方文档: https://docs.edennetwork.io/mechanism   flashbots 介绍: https://www.chainnews.com/zh-hant/articles/008263592610.htm   flashbots 官网文档: https://docs.flashbots.net/new-to-mev",
```
[the remark this answers is not in the document]
Comment (yingjingyang) [reply basic/38-Flashbots-and-Eden/package.json:4]: fixed

**Review by liberhe — DISMISSED** [review]

File: basic/38-Flashbots-and-Eden/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Flashbot & Eden
+
+// todo 
+Fix the issue of not work of test code  && split Eden to other directory
+
+## 参考链接  
+flashbots github: https://github.com/flashbots/pm#resources
```
Comment (luckyyang) [inline basic/38-Flashbots-and-Eden/README.md:7]: Should delete the spaces at the end of each line

Comment (yingjingyang) [reply basic/38-Flashbots-and-Eden/README.md:7]: Not the final version, the doc && the readme will be fixed soon

**Review by maroonstar — APPROVED** [review]

**Review by liberhe — APPROVED** [review]

**Pull request merged**